app/Console/Commands/UpdateUserSocialAccount.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use App\Models\User;
+use App\Traits\FormatSocialAccount;
+use Illuminate\Console\Command;
+
+final class UpdateUserSocialAccount extends Command
+{
+    use FormatSocialAccount;
+
+    protected $signature = 'lcm:update-user-social-account';
+
+    protected $description = 'Update user social account to normal format';
+
+    public function handle(): void
+    {
+        $this->info('Start updating users social account...');
+
+        foreach (User::verifiedUsers()->get() as $user) {
+            $this->info('Updating '.$user->username.'...');
+            $user->twitter_profile = $this->formatTwitterHandle($user->twitter_profile);
```

Comment:
Pourquoi tu ne fais pas un update directement avec tous les champs ? Et tu colles `$this->info(...)` avec la data. Il faut rajouter un saut de ligne